### ISSUE TITLE
Error on rendering localize files

### ISSUE THREAD

**Issue by sypets** [t3docs/docker-render-documentation#21]
If I use latest version of [sample extension manual](https://github.com/TYPO3-Documentation/TYPO3CMS-Example-ExtensionManual) and render locally, there are errors, but the rendering seems to work fine.

If the directories **Documentation/Localization.*tmpl** are removed, the errors disappear. Also, if the localization directories are renamed to their final name (without tmpl), the errors also disappear.

We could easily solve this now in the sample extension manual by moving the localization template dirs to a location outside of Documentation or renaming them, but as this manual has been this way forever, I thought I would ask here first.

output:

```
==================================================
   14-Provide-the-project/40-Steps-B/run_28-Move-and-choose-masterdoc.py
Traceback (most recent call last):
  File "/ALL/Toolchains/RenderDocumentation/14-Provide-the-project/40-Steps-B/run_28-Move-and-choose-masterdoc.py", line 143, in <module>
    mdfile = os.path.join(TheProject, left, locale_folders[localization], right)
KeyError: u'fr_FR.tmpl'
   exitcode: 1

==================================================
   16-Convert-and-fix-and-check/run_00-Check-requirements.py
   exitcode: 22

```

```
We saw these exitcodes (code, count):
{
  "0": 43,
  "1": 1,
  "22": 9
}
```

**Comment by sypets** [comment]
For now, I moved the localization directories to parent directory in order to be able to render the documentation.

**Comment by sypets** [comment]
It is not possible to render the [sphinx](https://github.com/TYPO3-extensions/sphinx) extension with the Docker container.

But that is also due to another issue: https://github.com/xperseguers/t3ext-sphinx/issues/9

But, AFAIK, there is still a general problem with rendering documentation which contains localization.

**Comment by sypets** [comment]
Please add "bug" label.

**Comment by marble** [comment]
① New version master >= v2.0.0 is working without error.
![grafik](https://user-images.githubusercontent.com/307057/58380024-05356a80-7fac-11e9-8993-b5ea929929aa.png)

② Old version v1.6.x still needs to be tested. Initially it had been tested with Sphinx and worked back then as well.

**Comment by marble** [comment]
This has two reasons:

① The sample extension manual has had an illegal structure. You already fixed that. Anyway, it's been a "bug" of that manual.

② It's a followup error of #52. This has already been fixed for >= v2.0.0.

So I'm closing this issue here.

**Comment by sypets** [comment]
@marble I don't know enough about rendering other languages. It would be good to have a testcase with localization which "should work" (e.g. a minimal sample manual with localization).

The [sample extension manual](https://github.com/TYPO3-Documentation/TYPO3CMS-Example-ExtensionManual) is not a good example because localization has been deactivated entirely (by renaming the localization directories, e.g. `Localization.de_DE.tmpl`). I wasn't aware of it having an illegal structure. I thought rendering the localization just didn't work. 

I tested rendering a manual with localization with the "sphinx" extension again and it works. 

I tested this way:

1. Get [current sphinx extension](https://github.com/TYPO3-extensions/sphinx). 
2. Try to render. Fails because of #52
3. Remove `AdvancedUsersManual/Plugins`, remove from toctree in `Documentation/AdvancedUsersManual/Index.rst`, remove plugins from Settings.cfg, etc. (to test without #52 error) - see [my branch test-docker-rendering](https://github.com/sypets/sphinx/tree/sypets/test-docker-rendering)
4. try to render again: rm Documentation-GENERATED and render from scratch 
5. works

(The last time I tested this, I had neglected to make the necessary changes in the `Localization.fr_FR` directory as well. That's apparently why I still got errors).
